packages/expo-video/android/src/main/java/expo/modules/video/VideoPlayer.kt

```diff
@@ -113,7 +114,9 @@ class VideoPlayer(val context: Context, appContext: AppContext, source: VideoSou
     }
 
     override fun onVolumeChanged(volume: Float) {
-      this@VideoPlayer.volume = volume
+      if (!muted) {
```

Comment:
Is this the correct behavior? My expection would be that mute gets disabled when volume is changed.